src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp.mdx

```diff
@@ -0,0 +1,283 @@
+
+If using the collector, prefer the [otlphttpexporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter).
+
+<table>
+  <thead>
+    <tr>
+      <th>
+        Environment
+      </th>
+
+      <th>
+        gRPC
+      </th>
+
+      <th>
+        HTTP
+      </th>
+
+      <th>
+        Endpoint
+      </th>
+
+      <th>
+        Supported Ports
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        US OTLP
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        `https://otlp.nr-data.net`
+      </td>
+
+      <td>
+        `443`, `4317`, `4318`
+      </td>
+
+    </tr>
+
+    <tr>
+      <td>
+        EU OTLP
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        `https://otlp.eu01.nr-data.net`
+      </td>
+
+      <td>
+        `443`, `4317`, `4318`
+      </td>
+
+    </tr>
+
+    <tr>
+      <td>
+        US FedRAMP OTLP<br/>(See [FedRAMP compliance](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#otlp-api) for more information)
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        `https://gov-otlp.nr-data.net`
+      </td>
+
+      <td>
+        `443`, `4317`, `4318`
+      </td>
+
+    </tr>
+
+    <tr>
+      <td>
+        Infinite Tracing<br/>(See [best practices](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts/#infinite-tracing)  for endpoint details
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        `https://{trace-observer}`
+      </td>
+
+      <td>
+        `443`
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="note-ports"
+    title="Why three ports?"
+  >
+    OTLP standards designate gRPC traffic to port `4317`, and HTTP traffic to port `4318`. The New Relic US FedRamp Native OTLP endpoint adheres to those specifications. It also allows both gRPC and HTTP traffic on port `443`. For simplicity, you can omit the port since the `https` scheme automatically selects port `443`.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="note-endpoints"
+    title="Additional endpoint details"
+  >
+    Per the [OpenTelemetry spec](https://opentelemetry.io/docs/specs/otel/protocol/exporter/#endpoint-urls-for-otlphttp) on endpoint URLs for OTLP/HTTP&#x3A; If you are sending OTLP/HTTP traffic and using the signal agnostic environment variable (`OTEL_EXPORTER_OTLP_ENDPOINT`), you can simply set `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:{port}` and the exporter should append the appropriate path for the signal type (i.e., `v1/traces` or `v1/metrics`).
```

Comment:
```suggestion
    Per the [OpenTelemetry spec](https://opentelemetry.io/docs/specs/otel/protocol/exporter/#endpoint-urls-for-otlphttp) on endpoint URLs for `OTLP/HTTP&#x3A`. If you are sending OTLP/HTTP traffic and using the signal agnostic environment variable (`OTEL_EXPORTER_OTLP_ENDPOINT`), you can simply set `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:{port}` and the exporter should append the appropriate path for the signal type (i.e., `v1/traces` or `v1/metrics`).
```

@jack-berg Is the `;` supposed to be included?

Reply:
This renders to "on endpoint URLs for OTLP/HTTP: ..."

I repurposed this from somewhere else, but I reworked the punctuation to be less confusing.